### ISSUE TITLE
Remove Chrome 79 bug workaround for Sandcastle

### ISSUE THREAD

**Issue by mramato** [CesiumGS/cesium#8460]
Sandcastle no longer works in Chrome 79 released earlier today  (at least on Linux, I haven't tried other OS's).

At first I thought this was local Sandcastle only, but if you start sandcastle.cesium.com with a clear cache, it just prints `Error reading source file: Hello World` in the console. You can see a similar error locally if you navigate to demos after the initial one fails to load.

**Comment by mramato** [comment]
Update: I'm on a flaky hotel connection so it's possible that sandcastle.cesium.com is actually working, but local is definitely broken.  (Or perhaps it's a race condition).

**Comment by IanLilleyT** [comment]
In local sandcastle I get: `Refused to apply style from 'http://localhost:8080/Apps/templates/bucket.css' because its MIME type ('text/html') is not a supported stylesheet MIME type, and strict MIME checking is enabled.`

But the bucket.css file lives in `Apps/Sandcastle/templates/`, not `Apps/templates`

Hope this helps

**Comment by OmarShehata** [comment]
I think the behavior of `DOMParser` has changed between Chrome 78 -> 79. It seems to have gone from sync to async. The code below parses an HTML doc from a string, and attempts to extract the script tag. In Chrome 78, this prints out the script tag successfully. In 79 it outputs null. Adding a setTimeout delay before checking for the script tag shows it just takes a bit longer now.

```
var parser = new DOMParser();
var doc = parser.parseFromString(`<!DOCTYPE html>
<html lang="en">
<body>
<style>
    @import url(https://cesium.com/downloads/cesiumjs/releases/1.64/Build/Cesium/Widgets/widgets.css);
</style>
<script id="cesium_sandcastle_script">
</script>
</body>
</html>`, 'text/html');
doc.querySelector('script[id="cesium_sandcastle_script"]');
```

Here's where this happens in Sandcastle:

https://github.com/AnalyticalGraphicsInc/cesium/blob/2c488cc62108cd982b9289b22dfd2bc63dd4b17d/Apps/Sandcastle/CesiumSandcastle.js#L739-L745

If you add a setTimeout here on line 742 and everything after in that block, Sandcastle will work (although you still get some CSS errors, I think because other parts calling the same function have the same issue).

**Comment by OmarShehata** [comment]
It's possible DOMParser was always async and it was just fast enough that we never noticed. The sample code I have up there is a minimal example that reproduces the issue. Removing the `@import` line makes it work fine - I'm assuming that somehow delays the parsing (but only in Chrome 79) and causes it to not have the document fully parsed/ready by the next line of code.

I don't see in the spec anything about whether it should be sync/async: https://w3c.github.io/DOM-Parsing/#the-domparser-interface

**Comment by OmarShehata** [comment]
Created a Chromium issue here: https://bugs.chromium.org/p/chromium/issues/detail?id=1033781&can=2&q=DomParser#makechanges

**Comment by mramato** [comment]
Thanks again @OmarShehata we deployed a temporary fix with #8466. Looks like Chrome will have the issue fixed in Chrome 80, so once that's out we can undo #8466.
